src/oas.md

```diff
@@ -4012,257 +3285,996 @@ This object MAY be extended with [Specification Extensions](#specification-exten
+          $dynamicAnchor: generic-array
+          type: number
+    stringArray:
+      $id: array_of_strings
+      $ref: fully_generic_array
+      $defs:
+        stringsOnly:
+          $dynamicAnchor: generic-array
+          type: string
+    objWithTypedArray:
+      $id: obj_with_typed_array
+      type: object
+      required:
+      - dataType
+      - data
+      properties:
+        dataType:
+          enum:
+          - string
+          - number
+      oneOf:
+      - properties:
+          dataType:
+            const: string
+          data:
+            $ref: array_of_strings
+      - properties:
+          dataType:
+            const: number
+          data:
+            $ref: array_of_numbers
 ```
 
-###### MutualTLS Example
+#### Annotated Enumerations
 
-```json
-{
-  "type": "mutualTLS",
-  "description": "Cert must be signed by example.com CA"
-}
-```
+The Schema Object's `enum` keyword does not allow associating descriptions or other information with individual values.
 
-```yaml
-type: mutualTLS
-description: Cert must be signed by example.com CA
-```
+Implementations MAY support recognizing a `oneOf` or `anyOf` where each subschema in the keyword's array consists of a `const` keyword and annotations such as `title` or `description` as an enumerated type with additional information. The exact behavior of this pattern beyond what is required by JSON Schema is implementation-defined.
 
-###### Implicit OAuth2 Example
+##### Model with Annotated Enumeration
 
 ```json
 {
-  "type": "oauth2",
-  "flows": {
-    "implicit": {
-      "authorizationUrl": "https://example.com/api/oauth/dialog",
-      "scopes": {
-        "write:pets": "modify pets in your account",
-        "read:pets": "read your pets"
-      }
+  "oneOf": [
+    {
+      "const": "RGB",
+      "title": "Red, Green, Blue",
+      "description": "Specify colors with the red, green, and blue additive color model"
+    },
+    {
+      "const": "CMYK",
+      "title": "Cyan, Magenta, Yellow, Black",
+      "description": "Specify colors with the cyan, magenta, yellow, and black subtractive color model"
     }
-  }
+  ]
 }
 ```
 
 ```yaml
-type: oauth2
-flows:
-  implicit:
-    authorizationUrl: https://example.com/api/oauth/dialog
-    scopes:
-      write:pets: modify pets in your account
-      read:pets: read your pets
+oneOf:
+  - const: RGB
+    title: Red, Green, Blue
+    description: Specify colors with the red, green, and blue additive color model
+  - const: CMYK
+    title: Cyan, Magenta, Yellow, Black
+    description: Specify colors with the cyan, magenta, yellow, and black subtractive color model
 ```
 
-#### OAuth Flows Object
-
-Allows configuration of the supported OAuth Flows.
+### Modeling Non-JSON Data
 
-##### Fixed Fields
+This section describes how to model non-JSON formats, either by mapping the format into the
+[JSON Schema data model](https://www.ietf.org/archive/id/draft-bhutton-json-schema-01.html#name-instance-data-model)
```

Comment:
The preferred URL pattern seems to be
```suggestion
[JSON Schema data model](https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-01.html#name-instance-data-model)
```

Reply:
@ralfhandl I copied it from elsewhere in the text, we can audit it separately.  This PR will not be merged so fixing these things is not relevant here.